examples/blogs__app-decorator/README.md

```diff
@@ -0,0 +1,38 @@
+# App decorator
+> Using class decorator to automatically attach instance to window
+
+The application source code uses a [class decorator](https://www.sitepoint.com/javascript-decorators-what-they-are/) to automatically save created instances on the `window` object when running inside Cypress.
```

Comment:
Some inconsistency on the naming - app decorators vs class decorators. I think they're just called 'decorators' or 'JavaScript decorators' typically. I'd stick to the most used terminology for better find-ability. 

I don't see the use of 'App decorators' in google search at all and a few mentions of 'class decorators' although those span across languages outside of JavaScript.

Reply:
Yeah, it is kind of mixing: so this is class decorator that allows the user to expose an instance to do "app action". Let me just use JavaScript / class decorator for consistency.

Reply:
I will rename the recipe